src/languages/de-DE.json

```
@@ -0,0 +1,278 @@
+	"global_cannot_action_role_equal_bot": "{emote_warn} Du kannst diese Aktion nicht an **{user_tag}** durchführen, da meine Rolle niedriger ist als die Rolle vom User!",
+	"global_cannot_action_role_higher": "{emote_warn} Du kannst diese Aktion nicht an **{user_tag}** durchführen, Da er eine höhere Rolle als du hast!",
+	"global_cannot_action_role_higher_bot": "{emote_warn} Du kannst diese Aktion nicht an **{user_tag}** durchführen, da er eine Höhere Rolle als ich hat!",
+	"global_cannot_action_bot_self": "{emote_warn} You cannot perform this action on the bot!",
+
+	"premium_message": "Das ist ein Premium Befehl, Werde Patreon und bekomme Zugriff auf die Premium Befhele <https://www.patreon.com/bulbbot>",
+
+	"event_message_args_unexpected": "{emote_warn} Unerwartetes Argument `{arg}`. Erwartet `{arg_expected}` bekommen `{arg_provided}`\n{emote_wrench} **Korrekte Benutzung:** {usage}",
+	"event_message_args_missing": "{emote_warn} Fehlendes Argument `{arg}`. Erwartet `{arg_expected}` bekommen `{arg_provided}`\n{emote_wrench} **Korrekte Benutzung:** {usage}",
+	"event_message_args_unexpected_list": "{emote_warn} Kann nicht analysieren `{arg}` als `{arg_expected}`.\n{emote_wrench} **Richtige Benutzung:** {usage}",
+	"event_message_args_missing_list": "{emote_warn} Fehlende argumente `{arg}`. Erwartet `{arg_expected}` bekommen `{arg_provided}`\n{emote_wrench} **Richtige Benutzung:** {usage}",
+	"event_message_edit": "{emote_edit} Nachricht von **{target_tag}** `{target_id}` wurde Bearbeitet in <#{channel_id}>\n`ID (channel-message): {after_channel_id}-{after_id}`\n**B:** {before}\n**A:** {after}",
+	"event_message_delete": "{emote_trash} Nachricht von **{target_tag}** `{target_id}` wurde gelöscht in <#{channel_id}>\n`ID (channel-message): {after_channel_id}-${after_id}`\n**C:** {content} \n**A:** {attachment}\n**E:** {embed}",
+
+	"event_channel_create": "**Kanal erstellt:** <#{channel_id}> `({channel_type})`",
+	"event_channel_delete": "**Kanal geköscht:** #{channel_name} `({channel_id})` `({channel_type})`",
+	"event_channel_update_topic_added": "<#{channel_id}> Kanal Thema wurde hinzugefügt `{channel_topic}`\n",
+	"event_channel_update_topic_removed": "<#{channel_id}> Kanal Thema wurde entfernt\n",
+	"event_channel_update_topic_updated": "<#{channel_id}> Kanal Thema wurde von `{oldchannel_topic}` zu `{newchannel_topic} geändert`\n",
+	"event_channel_update_name": "<#{channel_id}> Name wurde von `#{oldchannel_name}` zu `#{newchannel_name} geändert`\n",
+	"event_channel_update_type": "<#{channel_id}> Art wurde von `{oldchannel_type}` zu `{newchannel_type} geändert`\n",
+	"event_channel_update_nsfw_enabled": "<#{channel_id}> nsfw wurde `aktiviert`\n",
+  "event_channel_update_nsfw_disabled": "<#{channel_id}> nsfw wurde `deaktiviert`\n",
+
+	"event_member_joined": "{emote_join} **Neues Mitglied:** **{user_tag}** `({user_id})` - **Account erstellt:** {user_age}",
```

Continuity to below translation.
```suggestion
	"event_member_joined": "{emote_join} **Mitglied ist dem Server beigetreten:** **{user_tag}** `({user_id})` - **Account erstellt:** {user_age}",
```